build/common.sh

```diff
@@ -171,6 +171,10 @@ function kube::build::verify_prereqs() {
 function kube::build::docker_available_on_osx() {
   if [[ -z "${DOCKER_HOST}" ]]; then
     kube::log::status "No docker host is set. Checking options for setting one..."
+    if [[ -a "/Applications/Docker.app/Contents/MacOS/Docker" ]]; then
```

Comment:
* Shouldn't this be moved above the status line?
* An alternative (and more consistent) way to check would be to look for the existence of `/var/run/docker.sock`.  This is the domain socket that docker hits by default with the latest version.  If that exists it is a good indication that docker for mac is working.